plugins/Unity/Launcher/xdgwatcher.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2019 UBports project.
+ * Author(s): Marius Gripsgard <marius@ubports.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "xdgwatcher.h"
+#include <QStandardPaths>
+#include <QDebug>
+#include <QDir>
+#include <QFileInfo>
+#include <QFile>
+#include <QTextStream>
```

Comment:
Please add a space after the `xdgwatcher.h` include, and sort the other ones alphabetically (nitpicking, I know :-) )